main.tf

```diff
@@ -180,21 +180,23 @@ module "athene2" {
 }
 
 module "kpi" {
-  source = "github.com/serlo/infrastructure-modules-kpi.git//kpi?ref=v1.0.0"
+  source = "github.com/serlo/infrastructure-modules-kpi.git//kpi?ref=v1.1.0"
   domain = local.domain
 
   grafana_admin_password = var.kpi_grafana_admin_password
+  grafana_serlo_password = var.kpi_grafana_serlo_password
 
+  athene2_database_username_readonly = "root"
```

Comment:
Normally the readonly user should be "serlo_readonly"? There's no root-user configured.